src/session.js

```diff
@@ -103,9 +107,9 @@ ghostdriver.Session = function(desiredCapabilities) {
     // Interesting details here: {@link http://stackoverflow.com/a/4995054}.
     _max32bitInt = Math.pow(2, 31) -1,      //< Max 32bit Int
     _timeouts = {
-        "script"            : _max32bitInt,
-        "implicit"          : 200,          //< 200ms
-        "page load"         : _max32bitInt,
+        "script"            : 30000,
```

Comment:
Mmm, the default value of those timeouts is a really delicate matter.

You sure about those changes?
Last time I checked this (consulting with various Selenium maintainers) is a good idea to be super-relaxed with default values, and that's why the "script" and "page load" were essentially unbounded. "implicit" was to make our client internal code wait "a bit" (you can see it's usage around).

Changing those values is a great responsibility you are taking: are you 100% sure?